opensrp-app/src/test/java/org/smartregister/repository/LocationRepositoryTest.java

```diff
@@ -245,6 +247,18 @@ public void testGetLocationsByTagName() {
         assertEquals(1, tags.size());
     }
 
+    @Test
+    public void testMarkLocationsAsSynced(){
```

Comment:
KIndly update the test name to match convention test[MethodName][ExpectedResult][Parameters] (Parameters optional if not present).